tdrs-frontend/.env.development

```diff
@@ -1,7 +1,7 @@
 REACT_APP_BACKEND_URL=http://localhost:8080/v1
```

Comment:
We discovered that `.env` files won't perform calculations so we had to put in the static value

Reply:
Ah, that makes sense! 

Would code comments be helpful here so that future viewers of the file don't have to do the math?

Reply:
@alexsoble I just updated the file with comments to clarify. I think this is a good idea.